build/ProductAssemblyInfo.cs

```diff
@@ -9,14 +9,16 @@
 using System.Runtime.CompilerServices;
 
 [assembly: AssemblyCopyright (
-    "Copyright 2016-2017 Microsoft. All rights reserved.\n" +
+    "Copyright 2016-2018 Microsoft. All rights reserved.\n" +
     "Copyright 2014-2016 Xamarin Inc. All rights reserved.")]
 [assembly: AssemblyVersion ("1.0.0.0")]
 
 // Spells 0.A.F.V, a magic version that UpdateBuildInfo will
 // replace with a real file version (see BuildInfoTask).
 [assembly: AssemblyFileVersion ("0.65.70.86")]
 
+#if !PRODUCT_ASSEMBLY_INFO_OMIT_IVT
+
 #if PCL_FACADE_ASSEMBLY
```

Comment:
the PCL_FACADE_ASSEMBLY section can go away, btw

Reply:
Ah, right. Nice. Also for future reference: when the rest of our non-SDK-style projects can become SDK-style projects, this whole file can go away and be handled entirely via MSBuild within `Directory.Build.props`. It's magic.